modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/object-fields/8-mdf-request.json

```diff
@@ -392,6 +392,32 @@
 			"required": false,
 			"type": "String"
 		},
+		{
+			"DBType": "BigDecimal",
+			"businessType": "PrecisionDecimal",
+			"externalReferenceCode": "MDF-REQUEST-CUR-EXCH-RATE",
```

Comment:
It feels off to store the exchange rate on the mdf request itself. Since an exchange rate is a globally applied thing that changes, if we need to store it probably it should be in a separate object that we can keep in sync in one place and reference, instead of on every instance of an mdf request

Reply:
We need to store the exchange rate because we will need this value for mdf claim amount conversion. It was aligned with Caio Monteiro that we would use a fixed exchange rate (from the moment that the mdf request was submitted) for all mdf related operations, so in case a user create a mdf claim 2 days after mdf request submission (for example) the converted amounts will be consistent.

Reply:
Yes, we are getting the exchange rate from commerce's currency api and simply dividing the totalRequestAmount (not sure if it's the exact name) by the exchange rate. We don't have this field in SF, but since it's just for use in the mdf chart in the home page we are doing it "locally".